action.yaml

```diff
@@ -34,9 +34,16 @@ runs:
         project_name=${repository:$repository_characters_to_skip}
 
         # Determine the target path
-        short_sha=${sha:0:8} # First 8 characters of the commit sha
+        short_sha=$(git rev-parse --short HEAD)
         test_result_target_path="s3://${REGULATORY_S3_BUCKET}/projects/${project_name}/commits/${short_sha}/test-results.xml"
-        
+
+        date=$(date -u +"%Y-%m-%dT%H:%M:%SZ")
```

Comment:
(minor) Might be nice to have description above this section explaining what this does. And/or an extra bullet point in the readme